src/o365/graph/commands/teams/teams-app-update.ts

```diff
@@ -0,0 +1,132 @@
+import * as request from 'request-promise-native';
+import * as fs from 'fs';
+import * as path from 'path';
+import auth from '../../GraphAuth';
+import Utils from '../../../../Utils';
+import config from '../../../../config';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import { CommandOption, CommandValidate } from '../../../../Command';
+import { Team } from './Team';
+import { GraphItemsListCommand } from '../GraphItemsListCommand';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  id: string;
+  filePath: string;
+}
+
+class TeamsAppUpdateCommand extends GraphItemsListCommand<Team> {
```

Comment:
Since this command is not listing any items, it should inherit from the GraphCommand base class instead.